src/equations/acoustic_perturbation_2d.jl

```diff
@@ -0,0 +1,153 @@
+@doc raw"""
+    AcousticPerturbationEquations2D(v_mean, rho_mean, c_mean)
+
+Acoustic perturbation equations in two space dimensions with constant mean flow. The equations are
+based on the APE-4 system introduced by Ewert and Schröder
```

Comment:
Maybe also clarify that `v_mean` is expected to be a tuple or SVector of size 2 while the other two are scalars.